debugtools/DDR_VM/test.xml

```diff
@@ -81,7 +81,7 @@ TCK - runs the DTFJ TCK
 			<entry key="tck.dumpcreator.class" value="ibm.j9.dumpcreators.SystemDumpCreator"/>
 			<entry key="tck.imagecreator.class" value="com.ibm.j9ddr.dtfj.tck.ImageCreator"/>
 			<entry key="tck.dumpcreator.launcher" value="${sdk.executable}"/>
-			<entry key="env.IBM_JAVA_OPTIONS" value="-Xdump:system:defaults:file=&quot;${tck.core.file}&quot;"/>
+			<entry key="env.OPENJ9_JAVA_VERSION" value="-Xdump:system:defaults:file=&quot;${tck.core.file}&quot;"/>
```

Comment:
@llxia @smlambert Anything that we need to be aware of when making this change?  Are there other consumers of this data that may be effected by this?

Reply:
- Should we change to `OPENJ9_JAVA_OPTIONS`?
- fyi. this test is not in our auto builds. From a quick look, I suspect we cannot run this test at all as DTFJ_TCK does not exist in openj9 https://github.com/eclipse/openj9/blob/99f226c95eb864ba5fbf3a6067e4b4dfa266efed/debugtools/DDR_VM/test.xml#L60-L66

Reply:
Thanks @llxia.  @pdbain-ibm Can you revert this change and open an issue for getting the DTFJ_TCKs into OpenJ9?

Reply:
Done: https://github.com/eclipse/openj9/issues/3654.
@llxia FYI.